yw_website/yw_website/urls.py

```diff
@@ -44,8 +44,8 @@
     ###
     path('', yw_website_views.home, name='home'),
     path('my_workflows/', yw_website_views.my_workflows, name='my_workflows'),
-    path('detailed_workflow/<int:document_id>/', yw_website_views.detailed_workflow, name='detailed_workflow'),
-    path('run_detail/', yw_website_views.run_detail, name='run_detail'),
+    path('detailed_workflow/<int:workflow_id>/<int:version_id>/', yw_website_views.detailed_workflow, name='detailed_workflow'),
```

Comment:
a better way to format this route would be `detailed_workflow/<int:workflow_id>/version/<int:version_id>/`